code/notificationbuilder/src/main/java/com/adobe/marketing/mobile/notificationbuilder/internal/builders/BasicNotificationBuilder.kt

```diff
@@ -66,7 +66,7 @@ internal object BasicNotificationBuilder {
         )
 
         // set the image on the notification
-        expandedLayout.setRemoteImage(
+        expandedLayout.setRemoteViewImage(
```

Comment:
`setRemoteImage` was not setting the view visibility to `GONE` in case of empty or null uri, changed it to use `setRemoteViewImage`